### ISSUE TITLE
Selective training of Convolution filters based on inputs (output value)

### ISSUE THREAD

**Issue by mustgoplay** [fchollet/keras#4550]
If I have a convnet with 30 filters, I want to be able to only train certain ones for a given input.  For example I may only want to train the first 5 filters and not all 30 of them.  How can I turn off learning for a given filter based on a given input (actually the value of the output for an input)?

I know that I will need a custom layer and think I need a batch size of 1 but I'm not sure about the latter.  From what I could figure out so far it just seems like either all the weights are trainable or they're all not and you can't pick and choose a specific filter to train.  Maybe with a mask but then I don't know if that's applicable by filter or just by weight value.

Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
